ocaml/xapi/dbsync.ml

```diff
@@ -70,7 +70,9 @@ let update_env ~__context =
          	  been set on the pool record before we run it [otherwise we
          	  try and sync config files from the old master if someone's
          	  done a pool.designate_new_master!] *)
-      if not (Pool_role.is_master ()) then resync_dom0_config_files ()
+      if not (Pool_role.is_master ()) then resync_dom0_config_files () ;
+      if Pool_role.is_master () then
+        Dbsync_master.update_pool_recommendations ~__context
```

Comment:
Please add the call to `update_pool_recommendations` to `update_env` in dbsync_master.ml itself.